data_diff/dbt.py

```diff
@@ -165,10 +170,13 @@ def dbt_diff(
                         + "Skipped due to unknown primary key. Add uniqueness tests, meta, or tags.\n"
                     )
 
-        # wait for all threads
-        if diff_threads:
-            for thread in diff_threads:
-                thread.join()
+    for future in as_completed(futures):
```

Comment:
Example Error log
![image](https://github.com/datafold/data-diff/assets/19176976/bc7796a7-84a0-4955-9700-fe5c4da1f708)

Reply:
I removed this as I realized it's too noisy to be helpful and that a user can just as easily add the `--debug` flag to figure out the problem.

![image](https://github.com/datafold/data-diff/assets/19176976/63e43ae1-a0b2-419f-b070-890a14905409)

![image](https://github.com/datafold/data-diff/assets/19176976/46ea42ae-b692-4b5a-b567-d4c9bffce592)